apps/meteor/tests/end-to-end/api/18-oauthapps.js

```diff
@@ -161,4 +161,90 @@ describe('[OAuthApps]', function () {
 				});
 		});
 	});
+
+	describe('[/oauth-apps.update]', () => {
+		let appId;
+
+		before((done) => {
+			const name = 'test-oauth-app';
+			const redirectUri = 'https://test.com';
+			const active = true;
+			request
+				.post(api('oauth-apps.create'))
+				.set(credentials)
+				.send({
+					name,
+					redirectUri,
+					active,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.end((err, res) => {
+					appId = res.body.application._id;
+					done();
+				});
+		});
+
+		it('should update an app name, active and redirect URI correctly by its id', async () => {
```

Comment:
Maybe change this wording for something like:
'should update an app's name, its Active and Redirect URI fields correctly by its id.